lightly/transforms/amplitude_rescale_transform.py

```diff
@@ -0,0 +1,42 @@
+from typing import Tuple
+
+import numpy as np
+import torch
+from torch import Tensor
+
+
+class AmplitudeRescaleTranform:
+    """
+    This transform will rescale the amplitude of the Fourier Spectrum (`input`) of the image and return it.
+    The scaling value *p* will range within `[m, n)`
```

Comment:
Please follow the same docstring formatting as for other transforms: https://github.com/lightly-ai/lightly/blob/b8613dc6765e26e7bc19fb99797a26656e6e389d/lightly/transforms/rotation.py#L13-L31